src/emphasis/badges/Badges.tsx

```diff
@@ -11,9 +11,16 @@
 import * as React from "react";
 import {classNames} from "../../utils";
 
+/**
+ * Badge Props
+ * color: color of label
+ * status: status of label
+ * dataqa: quality engineering testing field
```

Comment:
use below format for documentation
@param {dataqa} quality engineering tag

Reply:
Addressed